merge/action.yml

```diff
@@ -36,6 +36,11 @@ inputs:
       If true, the artifacts that were merged will be deleted.
       If false, the artifacts will still exist.
     default: 'false'
+  include-hidden-files:
```

Comment:
Does it make sense for the `merge` sub-action to have this input too?

Reply:
More options are a good thing so I would steer towards having it available too.